File: Go/Fizzbuzz.go
// Fizzbuzz is usually the first program I write in any language and it was fun learning how to do it in Go!
package main

import (
	"fmt"
	"strconv"
)

func main() {
	for i := 1; i <= 100; i++ {
		output := "";
		if i % 3 == 0 {
			output += "Fizz";
		}
		if i % 5 == 0 {
			output += "Buzz";
		}
		if output == "" {
			output = strconv.FormatInt(int64(i), 10)
		}
		fmt.Println(output);
	}
}

File: Java/FizzBuzz.java
import java.util.stream.IntStream;

public class FizzBuzz{
    
    public static void main(String[] args){
        IntStream.rangeClosed(1, 100).mapToObj(i -> i % 5 == 0 ? i % 7 == 0 ? "FizzBuzz" : "Fizz" : i % 7 == 0 ? "Buzz" : i).forEach(System.out::println);
    }
}
